apps/shooting-clubs/acceptance/pages/club-secretary-email.js

```diff
@@ -1,5 +1,24 @@
 'use strict';
 
+let I;
 module.exports = {
-  url: 'club-secretary-email'
+  _init() {
+    I = require('so-acceptance/steps.js')();
+  },
+
+  url: 'club-secretary-email',
+  'club-secretary-email-id': '#club-secretary-email',
+  'club-secretary-phone-id': '#club-secretary-phone',
```

Comment:
if you put both of these in a separate fields object you can reference them easier in your tests with something like:
```
I.seeElements(Object.keys(page.fields));
```